pandas/__init__.py

```diff
@@ -49,7 +49,15 @@
 from pandas.tools.merge import (merge, ordered_merge,
                                 merge_ordered, merge_asof)
 from pandas.tools.pivot import pivot_table, crosstab
-from pandas.tools.plotting import scatter_matrix, plot_params
+
+# deprecate tools.plotting, plot_params and scatter_matrix on the top namespace
+import pandas.tools.plotting
```

Comment:
I think this line is just to make sure `pd.tools.plotting` is tab-completable/directly usable (without doing `from pandas.tools import plotting` as a user) for back-compat, so not related to `pandas.plotting`